ecosystem/sep-0024.md

```diff
@@ -330,8 +330,8 @@ Request Parameters:
 | `account`                     | `G...` or `M...` string | (optional) The Stellar or muxed account the client wants to use as the destination of the payment sent by the anchor. Defaults to the account authenticated via SEP-10 if not specified.                                                                                                                                                                                                                                         |
 | `memo_type`                   | string                  | (optional) Type of memo that anchor should attach to the Stellar transaction, one of `text`, `id` or `hash`.                                                                                                                                                                                                                                                                                                                     |
 | `memo`                        | string                  | (optional) Value of memo to attach to transaction, for `hash` this should be base64-encoded. Because a memo can be specified in the SEP-10 JWT for [Shared Accounts](#shared-omnibus-or-pooled-accounts), this field can be different than the value included in the SEP-10 JWT. For example, a client application could use the value passed for this parameter as a reference number used to match payments made to `account`. |
-| `wallet_name`                 | string                  | (optional) In communications / pages about the deposit, anchor should display the wallet name to the user to explain where funds are going.                                                                                                                                                                                                                                                                                      |
-| `wallet_url`                  | string                  | (optional) Anchor should link to this when notifying the user that the transaction has completed.                                                                                                                                                                                                                                                                                                                                |
+| `wallet_name`                 | string                  | (**deprecated**, optional) In communications / pages about the deposit, anchor should display the wallet name to the user to explain where funds are going. However, anchors should use client_domain (for custodial) and `sub` value of JWT (for non-custodial) to determine wallet information.                                                                                                                                |
```

Comment:
How does an anchor determine that the client is custodial or not? The text seems to imply they should choose client domain or sub based on whether the client is custodial.

Is the intent that the anchor use client domain if it is present, and otherwise fall back to sub? If so, I suggest this change:

```suggestion
| `wallet_name`                 | string                  | (**deprecated**, optional) In communications / pages about the deposit, anchor should display the wallet name to the user to explain where funds are going. However, anchors should use client_domain if present, otherwise `sub` value of JWT to determine wallet information.                                                                                                                                |
```

Reply:
I think the better approach is actually the other way around, where the anchor maintains a set of addresses that are known to be owned by custodial services, and if the account present in the `sub` value doesn't match any of those, it should be treated as noncustodial.

I actually think this change isn't accurate. If the wallet is custodial, it won't specify a client domain. This applies to the change made to the `wallet_url` description too.

Reply:
Do you think I should add it somewhere as a commentary for anchor? We do have it in the stellar-docs, but I think for Anchor not using the Platform, it could be a valuable information

Reply:
Updated description

Reply:
About this line:

> However, anchors should use client_domain (for custodial) and `sub` value of JWT (for non-custodial) to determine wallet information. 

I don't think this is accurate right? Custodial wallets won't be passing `client_domain` in the SEP-10 flow and it won't be in the JWT, that is for non-custodial wallets.

Reply:
Yes I accidentally reversed it